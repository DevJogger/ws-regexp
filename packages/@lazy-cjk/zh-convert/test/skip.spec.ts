import { cn2tw, tw2cn } from '../index';

describe(`skip`, () =>
{
	const input = '简繁转换功能测试簡繁轉換功能測試';
	const skip = '转换轉換';

	test(`cn2tw`, () =>
	{

		let actual = cn2tw(input, {
			skip,
		});

		expect(actual).not.toStrictEqual(input);

		skip
			.split('')
			.forEach(c => expect(actual).toContain(c))
		;

		expect(actual).toMatchSnapshot();

	});

	test(`tw2cn`, () =>
	{

		let actual = tw2cn(input, {
			skip,
		});

		expect(actual).not.toStrictEqual(input);

		skip
			.split('')
			.forEach(c => expect(actual).toContain(c))
		;

		expect(actual).toMatchSnapshot();

	});

})
