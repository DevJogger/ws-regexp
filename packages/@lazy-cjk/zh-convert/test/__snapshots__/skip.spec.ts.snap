// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`skip cn2tw 1`] = `"簡繁转换功能測試簡繁轉換功能測試"`;

exports[`skip tw2cn 1`] = `"简繁转换功能测试简繁轉換功能测试"`;
